packages/fxa-payments-server/src/lib/apiClient.ts

```diff
@@ -261,13 +261,16 @@ export async function apiCapturePaypalPayment(params: {
   idempotencyKey: string;
   priceId: string;
   token?: string;
+  promoCode?: string;
 }): Promise<{
   sourceCountry: string;
   subscription: Subscription;
 }> {
+  const { promoCode } = params;
```

Comment:
The endpoints are expecting `promotionCode`.

Reply:
fixed